app/models/app_feature_completion.rb

```diff
@@ -4,7 +4,7 @@ class AppFeatureCompletion < ActiveRecord::Base
 
   attribute :version, :app_version
 
-  validates :app, :version, presence: true
+  validates :version, presence: true
```

Comment:
Is this because `belongs_to :app` should already enforce presence?

Reply:
Yes, it is caught as a redundant validation for relationships.